gh-71189: Support all-but-last mode in os.path.realpath() (#117562)
python/cpython: Doc/library/os.path.rst

   re-raised.
   In particular, :exc:`FileNotFoundError` is raised if *path* does not exist,
   or another :exc:`OSError` if it is otherwise inaccessible.
   If *strict* is :data:`ALL_BUT_LAST`, the last component of the path

>    might be missing, but other errors are not ignored.

Reviewer: ```suggestion
   is allowed to be missing, but all other errors are raised.
```

Just for consistency with the rest of the doc
